apps/emqx_durable_storage/src/emqx_ds_replication_layer_meta.erl

```diff
@@ -86,15 +101,20 @@
 
 -record(?SHARD_TAB, {
     shard :: {emqx_ds:db(), emqx_ds_replication_layer:shard_id()},
+    %% Sites that currently contain the data:
+    replica_set :: [site()],
     %% Sites that should contain the data when the cluster is in the
     %% stable state (no nodes are being added or removed from it):
-    replica_set :: [site()],
+    target_set :: [site()] | undefined,
```

Comment:
Just checking: since the feature is not GA yet, we don't need to provide a table migration for this new field even if someone is experimenting with it, right?

Reply:
Good question. I assume we don't.